server/tools/attestation-service-ctl/src/management.rs

```diff
@@ -0,0 +1,114 @@
+use crate::management_api::Tee;
+use anyhow::{anyhow, Context, Result};
+use std::fs;
+use std::io::prelude::*;
+use std::path::Path;
+
+use crate::management_api::management_service_client::ManagementServiceClient;
+use crate::management_api::{
+    GetPolicyRequest, GetPolicyResponse, GetReferenceDataRequest, GetReferenceDataResponse,
+    RestoreDefaultPolicyRequest, RestoreDefaultReferenceDataRequest, SetPolicyRequest,
+    SetReferenceDataRequest,
+};
+
+pub const DEFAULT_MANAGEMENT_ADDR: &str = "127.0.0.1:3001";
+
+impl Tee {
+    fn from_str(tee: &str) -> Result<Tee> {
+        match tee {
+            "sgx" => Ok(Tee::Sgx),
+            "tdx" => Ok(Tee::Tdx),
+            "sevsnp" => Ok(Tee::SevSnp),
+            "sample" => Ok(Tee::Sample),
+            _ => Err(anyhow!("TEE: {} is not supported", tee)),
+        }
+    }
+}
+
+pub async fn set_policy_cmd(tee: &str, file: &Path, socket: &str) -> Result<()> {
+    let policy = fs::read_to_string(file).context(anyhow!("Read policy error"))?;
+
+    let request = SetPolicyRequest {
+        tee: Tee::from_str(tee)? as i32,
+        user: None,
+        content: policy.into_bytes(),
+    };
+
+    let mut client = ManagementServiceClient::connect(format!("http://{}", socket)).await?;
```

Comment:
Ideally we wouldn't have to do this same format every single time. Also, the variable `socket` here is more like an address than a socket.